packages/block-editor/src/components/use-setting/index.js

```diff
@@ -15,6 +15,8 @@ import { __EXPERIMENTAL_PATHS_WITH_MERGE as PATHS_WITH_MERGE } from '@wordpress/
 import { useBlockEditContext } from '../block-edit';
 import { store as blockEditorStore } from '../../store';
 
+const blockedPaths = [ 'color', 'typography', 'spacing' ];
```

Comment:
My first instinct would be to also block `border` and `layout` for consistency. However, `layout` is the only one that is actually accessed directly.

Reply:
I left `useSetting( 'layout' )` as it seems the use cases make sense. I removed the border.